### DISCUSSION TITLE
Is there a way to use ESLINT as a javascript module?

### DISCUSSION THREAD

**Discussion by suaha** [eslint/eslint#15372] [Q&A] [answered]
I want to run ESLINT in java (using Rhino or GraalJS), from what I learnt ESLINT require a nodejs environment but it is difficult to have it in any JAVA JS engine.

Is there a way to use ESLINT as a pure javascript module?

**Comment by nzakas** [comment, marked as answer]
No there isn’t. ESLint requires Node.js
